### PR TITLE
Fix azurecli profile

### PR DESCRIPTION
**Pull request by wturner123**
Updated documentation as per:
https://docs.microsoft.com/en-us/azure-stack/user/azure-stack-version-profiles-azurecli2?view=azs-1910#connect-to-azure-stack-hub

### CODE REVIEW

File: articles/azure/azs-how-create-spn-cli.md [hunk 1]
```diff
@@ -48,7 +48,10 @@ To log in and manage your resources via SPN you'll need to create an Azure appli
 
 - [Service Principal Name commands for 2018-03-01-hybrid profile](https://docs.microsoft.com/en-us/cli/azure/ad/sp?view=azure-cli-2018-03-01-hybrid)
 
-- [Create an Azure Service Principal Name with Azure CLI 2.0 for 2018-03-01-hybrid profile](azs-how-create-spn-cli.md)
+    > [!NOTE]
+    > There is currently no document for the 2019-03-01-hybrid profile.
+
+- [Create an Azure Service Principal Name with Azure CLI 2.0 for 2019-03-01-hybrid profile](azs-how-create-spn-cli.md)
```
Comment (dbrennand) [inline articles/azure/azs-how-create-spn-cli.md]: I don't understand this link here. It links to this document?

File: articles/azure/azs-how-create-spn-cli.md [hunk 2]
```diff
@@ -48,7 +48,10 @@ To log in and manage your resources via SPN you'll need to create an Azure appli
...
 
 ## Overview of the creation process for Azure Stack Hub SPN
```
Comment (dbrennand) [inline articles/azure/azs-how-create-spn-cli.md]: In this section, a `SPN Password` variable is provided but isn't used in the code?

Comment (dbrennand) [reply articles/azure/azs-how-create-spn-cli.md]: Fixed in commit 4d5ac62

File: articles/azure/azs-how-create-spn-cli.md (hunk 1 above)
Comment (wturner123) [reply articles/azure/azs-how-create-spn-cli.md]: Updated link in commit 75e756e

**Comment by cblackuk** [comment]
Very good changes, definitely needed the review :-)

**Pull request merged**